frontend/src/v5/ui/routes/viewer/viewer.tsx

```diff
@@ -30,11 +29,15 @@ import { SpinnerLoader } from '@controls/spinnerLoader';
 import { CentredContainer } from '@controls/centredContainer';
 import { TicketsCardViews } from './tickets/tickets.constants';
 import { ViewerCanvases } from '../dashboard/viewerCanvases/viewerCanvases.component';
+import { CalibrationContextComponent } from '../dashboard/projects/calibration/calibrationContext';
+import { ViewerGui } from '@/v4/routes/viewerGui';
+import { modelIsFederation } from '@/v5/store/tickets/tickets.helpers';
 
 export const Viewer = () => {
 	const [fetchPending, setFetchPending] = useState(true);
 
 	const { teamspace, containerOrFederation, project, revision } = useParams<ViewerParams>();
+	const isFed = modelIsFederation(containerOrFederation);
```

Comment:
Does this check make sense now that we are using query strings for the calibration mode?